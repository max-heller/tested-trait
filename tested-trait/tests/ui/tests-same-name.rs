use tested_trait::tested_trait;

fn main() {
    #[tested_trait]
    trait Foo {
        #[test]
        fn test() {}
        #[test]
        fn test() {}
    }
}
