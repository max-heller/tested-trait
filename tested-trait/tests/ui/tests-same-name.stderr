error: the test `test` is defined multiple times
 --> tests/ui/tests-same-name.rs:9:12
  |
9 |         fn test() {}
  |            ^^^^
